index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style/style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <title>Apple</title>
+</head>
+
+<body>
+  <nav class="nav-bar">
+    <button type="button" class="btn-item"><a href="#"><i class="fa fa-apple" aria-hidden="true"></i></a></button>
+    <button type="button" class="btn-item"><a href="#">Store</a></button>
+    <button type="button" class="btn-item"><a href="#">Mac</a></button>
+    <button type="button" class="btn-item"><a href="#">iPod</a></button>
+    <button type="button" class="btn-item"><a href="#">iPhone</a></button>
+    <button type="button" class="btn-item"><a href="#">iPad</a></button>
+    <button type="button" class="btn-item"><a href="">iTunes</a></button>
+    <button type="button" class="btn-item"><a href="#">Support</a></button>
+    <span class="search">
+      <i class="fa fa-search" aria-hidden="true"></i>
+      <input type="text" />
+    </span>
+  </nav>
+  <article class="main-section">
+    <img src="img/title.png" alt="" />
+    <img src="img/subhead.png" alt="" />
+    <div class="img3">
+      Watch the "Your Verse" films and explore the stories
+    </div>
+  </article>
+  <aside class="grid">
+    <a href="#"><img src="img/grid-1.jpg" alt="" /></a>
+    <a href="#"><img src="img/grid-2.jpg" alt="" /></a>
+    <a href="#"><img src="img/grid-3.jpg" alt="" /></a>
+    <a href="#"><img src="img/grid-4.jpg" alt="" /></a>
+  </aside>
```

Comment:
## Background

The main content should have an image set as background (as CSS [`background-image`](https://www.w3schools.com/cssref/pr_background-image.asp) or [`background -> url`](https://www.w3schools.com/cssref/css3_pr_background.asp)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_image_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients))

- [ ] Use `background-image: url()` rule for each image in this section